docs/development/core/development-functional-tests.asciidoc

```diff
@@ -277,47 +314,50 @@ await testSubjects.click(‘containerButton’);
 ** `testSubjects.click(testSubjectSelector)` - Click a test subject in the page; throw if it can't be found after some time
 
 **find:**:::
-* Source: {blob}test/functional/services/find.js[test/functional/services/find.js]
+* Source: {blob}test/functional/services/find.ts[test/functional/services/find.ts]
 * Helpers for `remote.findBy*` methods that log and manage timeouts
 * Popular methods:
 ** `find.byCssSelector()`
 ** `find.allByCssSelector()`
 
+**retry:**:::
+* Source: {blob}test/common/services/retry/retry.ts[test/common/services/retry/retry.ts]
+* Helpers for retrying operations
+* Popular methods:
+** `retry.try(fn, onFailureBlock)` - execute `fn` in a loop until it succeeds or the default try timeout elapses, optional `onFailureBlock` is executed before each retry attempt 
```

Comment:
Could we break this up? How about:

Execute `fn` in a loop until it succeeds or the default timeout elapses. The optional `onFailureBlock` is executed before each retry attempt.